addons/mail/static/src/components/message/message.scss

```diff
@@ -243,6 +243,12 @@
     &.o-has-message-selection:not(.o-selected) {
         opacity: 0.5;
     }
+
+    &.o-highlighted {
+        background-color: rgba($o-brand-primary, .1);
+        border-left: 4px solid $o-brand-primary;
+        padding-left: 2px;
+    }
```

Comment:
Thank you for the update. This works but I think it might be a bit too weak in that it could easily break without us noticing if we ever change the padding of message or something like that.

At minimum it should use variables, but I think a better alternative would be to introduce an actual DOM element for it, that is always taking the space, and just we change the color when we need to activate it.
This would be similar to `o_DiscussSidebarItem_activeIndicator`.